packages/slate-editor/src/modules/editor/getEnabledExtensions.ts

```diff
@@ -23,6 +23,7 @@ import { PasteSlateContentExtension } from '#extensions/paste-slate-content';
 import { PressContactsExtension } from '#extensions/press-contacts';
 import { SoftBreakExtension } from '#extensions/soft-break';
 import { StoryBookmarkExtension } from '#extensions/story-bookmark';
+import { SnippetExtension } from '#extensions/snippet';
```

Comment:
Please use plural for extension name and extension folder.

`SnippetsExtension` and `#extensions/snippets`.